### ISSUE TITLE
Redirect to a new page when context changes doesn't work

### ISSUE THREAD

**Issue by v3gard** [equinor/fusion-cli#868]
I'm no longer able to use the context selector to navigate a user to a certain page if I am using a page where the contextId AND an additional ID is in the URL. It works as long as the contextId is the only id in the path, e.g. `/{contextId}`

# Demo
![redirect_not_working](https://user-images.githubusercontent.com/343078/197175704-d305f830-024f-48ad-9f9d-758fe0e06aae.gif)

# Repro project
[myawesomeapp.zip](https://github.com/equinor/fusion-cli/files/9838208/myawesomeapp.zip)

**Comment by v3gard** [comment]
@odinr Forgot to mention that this is also a problem in production, and it affects the workflow of our users quite a bit. Is it possible to get an ETA on when a fix is available?

**Comment by odinr** [comment]
@v3gard could we take a meeting on that so i can see how you do the actual navigation?

**Comment by v3gard** [comment]
@odinr Yes. This is still an issue even though I updated to @equinor/fusion 3.4.0. You can look  at this application for reproduction. Are we doing something wrong on our end?

[myawesomeapp.zip](https://github.com/equinor/fusion-cli/files/9909501/myawesomeapp.zip)
